build_container.sh

```diff
@@ -14,6 +14,7 @@ DEBIAN_FRONTEND="noninteractive" apt-get install --no-install-recommends -y \
     autoconf autoconf-archive automake libtool \
     libclang-dev iproute2 \
     libasound2 libasound2-dev \
+    libepoxy libepoxy-dev \
```

Comment:
I think this needs to be libepoxy0 libepoxy-dev (https://packages.ubuntu.com/search?mode=exactfilename&suite=noble&section=all&arch=any&keywords=libepoxy&searchon=names)

Reply:
OK thanks!